### PR TITLE
Update the puppet icon(s)

### PR DESCRIPTION
**Pull request by dhollinger**
Updated the Puppet icon and color and added the same for Puppet .epp files﻿

This is dependent on the following PR in ryanoasis/nerd-fonts: https://github.com/ryanoasis/nerd-fonts/pull/652

### CODE REVIEW

**Review by kyazdani42 — APPROVED** [review]

**Comment by kyazdani42** [comment]
should we merge this now or wait for the related PR to be merged ?

**Comment by dhollinger** [comment]
I would wait until the related PR is merged. Otherwise, users will end up with a Horizontal line/rule instead of an icon.

**Comment by dhollinger** [comment]
@kyazdani42 https://github.com/ryanoasis/nerd-fonts/pull/652 was merged last month. I fixed the merge conflicts and this can be merged anytime you are ready

File: lua/nvim-web-devicons.lua [hunk 1]
```diff
@@ -1373,4 +1377,3 @@ return {
   has_loaded = function() return loaded end,
   get_icons = function() return icons end,
   set_up_highlights = set_up_highlights,
-}
```
Comment (kyazdani42) [inline lua/nvim-web-devicons.lua:1376]: just remove that diff and we'll be good to go then :)

Comment (dhollinger) [reply lua/nvim-web-devicons.lua]: Should be fixed now.

**Pull request merged**